Swift Vocabulary/Swift Vocabulary/DefinitionViewController.swift

```diff
@@ -10,23 +10,22 @@ import UIKit
 
 class DefinitionViewController: UIViewController {
 
+    var vocabWord: VocabularyWord?
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        updateViews()
         // Do any additional setup after loading the view.
     }
     
     @IBOutlet var definitionTextView: UITextView!
     
     @IBOutlet var wordLabel: UILabel!
-    /*
-    // MARK: - Navigation
-
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        // Get the new view controller using segue.destination.
-        // Pass the selected object to the new view controller.
-    }
-    */
-
-}
+    
+    func updateViews() {
+                if let vocabWord = vocabWord{
+                    title = vocabWord.word
+                    definitionTextView.text = vocabWord.definition
+                }
+            }
+        }
```

Comment:
Remember to highlight groups of code and press `CTRL + I` to indent everything correctly. Readability is very important :-)